CONTRIBUTING.md

```diff
@@ -825,6 +886,39 @@ The test executable will be placed into the source test directory and can be run
 qiskit-aer$ ./test/unitc_tests [Catch2-options]
 ```
 
+
+## Testing multiple-GPUs and/or multiple nodes
```

Comment:
I think we can move this section within the  `Building with MPI support` section.

Reply:
moved